### PR TITLE
Removing AWS from the collection and making it general

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Removing AWS from the collection and making it general

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,29 +2,21 @@
...
-* An AWS account with permissions on `ec2` on said regions with default profile in $HOME/.aws/credentials that will be used to provision ec2 compute nodes
-* A database service that can be accessed from the deployment regions; or otherwise, `rds` permissions on the AWS account so an Aurora service can be provisioned
+* Setup multi-region clusters infrastructure.
+* A database service that can be accessed from the deployment regions.
 * TLS certificates for the desired domain name to provide the authentication service
 
 ### 1. create ansible.cfg
 
 ```
-[defaults]
```
Comment (guidograzioli) [inline README.md:21]: let's keep this block, changing "ec2-user" to  "<ssh_user>"

File: galaxy.yml [hunk 2]
```diff
@@ -1,11 +1,12 @@
 ---
 namespace: infra
-name: rhbk_datagrid_aws
+name: rhbk-ha-cluster
```
Comment (guidograzioli) [inline galaxy.yml]: this should be `rhbk_ha_cluster` (no dash allowed in collection FQCN)

File: inventory/group_vars/eu_central_1.yml [hunk 3]
```diff
@@ -1,3 +1,3 @@
 ---
-aurora_endpoint: rhbk-ha-cluster.cluster-cqy2krcosm24.eu-central-1.rds.amazonaws.com:5432
+aurora_endpoint: rhbk-ha-cluster.cluster-cqy2krcosm24.eu-central-1:5432
```
Comment (guidograzioli) [inline inventory/group_vars/eu_central_1.yml]: because aurora is aws specific, we should rename to "db_" please

**Review by guidograzioli — CHANGES_REQUESTED** [review]
Please address comment; while you are here, can you also get rid of the "promtail" role and tasks?

**Comment by RanabirChakraborty** [comment]
@guidograzioli review now.

**Review by guidograzioli — APPROVED** [review]
LGTM

**Pull request merged**